components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIExportUtil.java

```diff
@@ -535,6 +539,16 @@ private static void exportMetaInformation(String archivePath, API apiToReturn, R
             //If a web socket API is exported, it does not contain a swagger file.
             //Therefore swagger export is only required for REST or SOAP based APIs
             if (!APIConstants.APITransportType.WS.toString().equalsIgnoreCase(apiToReturn.getType())) {
+                //For Graphql APIs, the graphql schema definition is exported along with the swagger and api definition.
+                //For Graphql APIs, the URI templates in the API object is not cleared as operation info are not
+                //in the swagger.
```

Comment:
In graphql, swagger has been used only for developer user to try out the operations defined at the schema. The URI templates in the API object has been clearly defined all the operations available at the schema definition.

Reply:
Yes.